docs/brand/colors.html

```diff
@@ -4,7 +4,7 @@
 description: See the <a href="/product/base/colors/">Product</a> section for colors when building with Stacks. While there is some cross-over with the Product colors, using Stacks color classes will mean they adjust depending on context (e.g., dark mode)
 ---
 
-<script src="{{ "/assets/js/feature.notices.js" | url }}" defer></script>
+<script src="{{ "/assets/js/entry.notices.js" | url }}" defer></script>
```

Comment:
```suggestion
<script src="{{ "/assets/dist/entry.notices.js" | url }}" defer></script>
```

Is copying over `assets/js` necessary for the compiled site? Might be worth ignoring that so that there's no question as to where to include the JS from.

Reply:
Nice catch! I think I straight up forgot to update this import and it just kept working so... See my comment below - I stopped copying over the assets folder in its entirety.